### DISCUSSION TITLE
Can pyscripter  add the memory function for “Open Remote File” ？

### DISCUSSION THREAD

**Discussion by wilikeit** [pyscripter/pyscripter#1326] [Ideas]
Menu→File→open remote file
I need to fill in the file path and ssh server each time. 
Can pyscripter automatically fill in the last content？
Automatically fill in the contents of last time, as below.

![image](https://github.com/user-attachments/assets/93c7388f-f548-4212-948e-1fe3b6e87e0f)
It would be better if file path provided a list of recently opened path histories to choose .

In addition, I discovered a phenomenon：
If I  open an existing file and click “Menu→File→save to remote，pyscripter    will create a new untitled file with the same content，It is not saved to a remote. If I  create a new untitled file, click ‘save to remote’，it will  save to remote. Is pyscripter designed that way?

**Comment by pyscripter** [comment]
https://github.com/pyscripter/pyscripter/issues/1329

**Reply by pyscripter** [reply]
Fixed in version control. The fix will be available with the next release.
